content/Lab1/10_Setup_Network_Resources/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Setup AWS Network Resources"
-weight: 30
+weight: 10
 ---
 
 In this section we will walk through the setup of networking resources required for this lab.
 
-[AWS Network Resources - Description](../30_Setup_Network_Resources/1_network_resources.md)
+[AWS Network Resources - Description](../10_Setup_Network_Resources/1_network_resources.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description